mobile_verifier/src/coverage.rs

```diff
@@ -531,8 +533,9 @@ impl CoveredHexes {
         {
             let hex = hex as u64;
             boosted |= boosted_hexes.is_boosted(&hex);
-            if indoor {
-                self.indoor
+            match (indoor, &radio_key) {
```

Comment:
possible suggestion to avoid repeated code:

```rust
if indoor {
   if matches!(radio_key, OwnedKeyType::Cbrs(_)) {
     &mut self.indoor_cbrs
   } else {
     &mut self.indoor_wifi
   }
   .entry(...)
   .or_default()
   .entry(signal_level)
   .or_default()
   .push(IndoorCoverageLevel { ... })
} else {
   if matches!(radio_key, OwnedKeyType::Cbrs(_)) {
      &mut self.outdoor_cbrs
   } else {
      &mut self.outdoor_wifi
   }
   .entry(...)
   /* etc */
}
```

Reply:
Addressed b7f66f3